src/Gemfile.lock

```diff
@@ -75,7 +75,7 @@ PATH
   specs:
     bosh-registry (0.0.0)
       aws-sdk
-      bosh_cpi
+      bosh_cpi (~> 2.0.6)
```

Comment:
I updated it, because there was "connected" error in unit test. But now I see that I had to fix unit test instead. Thank you for the good catch